docs/guide/api-environment-frameworks.md

```diff
@@ -98,15 +98,15 @@ const server = await createServer({
   server: { middlewareMode: true },
   appType: 'custom',
   environments: {
-    server: {
+    ssr: {
       // by default, modules are run in the same process as the vite server
     },
   },
 })
 
 // You might need to cast this to RunnableDevEnvironment in TypeScript or
 // use isRunnableDevEnvironment to guard the access to the runner
-const environment = server.environments.node
+const ssrEnvironment = server.environments.ssr
```

Comment:
I think it is ok to use `server` here to show that the name could be different than `ssr`. Maybe we could rename `server` to `viteServer` to and correct the `node` -> `server` typo.

Reply:
But an empty `server` will not do anything, will it? Considering that the comments are suggesting runnable dev environment, should I add configuration for that, and maybe some build configuration?

And it could be called `serverEnvironment` instead of just `server` to be more clear, but I could call it `server` if that's what you want.

Reply:
`server` by default would be a `RunnableDevEnvironment`. I don't know what you mean by empty.

Reply:
Wow, indeed, it even says so in the guide itself. I thought that `ssr` gets special treatment, but it's actually any non-`client` environment 😮